src/main/java/org/folio/circulation/support/http/server/ValidationError.java

```diff
@@ -10,24 +10,39 @@
 import io.vertx.core.json.JsonObject;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import org.folio.circulation.support.json.JsonPropertyWriter;
 
 @Getter
 @EqualsAndHashCode
 public class ValidationError {
   private final String message;
   private final Map<String, String> parameters;
+  private String code;
```

Comment:
Why not final?

Reply:
because there is constructor that is used without this field and I am not sure about put some default value into this field

Reply:
added